HelpSource/Classes/ControlRate.schelp

```diff
@@ -23,6 +23,40 @@ code::
 { ControlRate.ir.poll }.play;
 ::
 
+compare ControlRate (.kr) and link::Classes/SampleRate:: (.ar)
```

Comment:
Instead of multiplying arrays, you can also just have an array of ugens. Also, the plot is misleading, because of a glitch in the interpolation pf plot, at that size. So here is asuggestion:

```supercollider

(
{
	var freq = 400;
	
	[
	// Pulses at control rate:
		LFPulse.ar(ControlRate.ir, 0, 0.01, 2, -1),
	
	// Sinewave at control rate:
		SinOsc.kr(freq),
	
	// Pulses at sample rate:
		LFPulse.ar(SampleRate.ir/2, 0, 0.1, 2, -1),
	
	// Sinewave at sample rate:
		SinOsc.ar(freq)
	]
}.plot(0.01, bounds:700@300).plotMode_(\steps);
)
```

Reply:
thank you, that's much clearer. I'm now thinking if the LFPulse parts, that I added to somehow make marks, are really needed or successful in demonstrating the different rates.  Perhaps something like this is better:
```
(
{
	var freq = 400;
	
	[
	// Sinewave at control rate:
		SinOsc.kr(freq),
	
	// Sinewave at sample rate:
		SinOsc.ar(freq)
	]
}.plot(0.01, bounds:800@500);
)
```
?

Reply:
yes, that would be better!